### ISSUE TITLE
Library Request: PrettyTime NLP for natural language dates parsing

### ISSUE THREAD

**Issue by beetee17** [nus-cs2103-AY2223S1/forum#350]
## Library

[PrettyTime NLP](https://www.ocpsoft.org/prettytime/nlp/). [Github link](https://github.com/ocpsoft/prettytime)

## Purpose

For an improved user experience as it allows users to specify dates in plain English. We intend to use this for users to easily set deadlines for tasks without the need to follow specific date formats e.g. DD-MM-YYY etc.

## License

[Apache License 2.0](https://github.com/cbeust/jcommander/blob/master/license.txt)

**Comment by damithc** [comment]
Approved.
